DEVELOPMENT.md

```diff
@@ -0,0 +1,229 @@
+# Development
+
+M3 is a very active, large codebase. This guide is intended for all potential
+contributors, designed to make it easy to get started with contributing to M3.
+
+## Setup your development environment
+
+First, fork https://github.com/m3db/m3 to your own user or org account on Github.
+
+Then, clone your fork:
+
+```bash
```

Comment:
Is there any reason it switches from m3 to m3db here? I know we all need to have further discussions on naming, but it's an sudden change, without explanation.